<launch>
    <rosparam file="$(find vessel_pointcloud_scanner)/config/params.yaml" command="load"/>
    <node name="display_time" pkg="vessel_pointcloud_scanner" type="display_time.py" output="screen"/>
    <node name="detection" pkg="vessel_pointcloud_scanner" type="detection.py" output="screen"/>
    <node name="vessel_pointcloud_scanner_node" pkg="vessel_pointcloud_scanner" type="vessel_pointcloud_scanner_node" output="screen"/> 
</launch>